tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -4,18 +4,19 @@
 
 from tdpservice.data_files.errors import ImmutabilityError
 from tdpservice.data_files.models import DataFile
-from tdpservice.data_files.validators import validate_data_file
+from tdpservice.data_files.validators import (
+    validate_file_extension,
+    validate_file_infection,
+)
+from tdpservice.security.models import ClamAVFileScan
 from tdpservice.stts.models import STT
 from tdpservice.users.models import User
 
 
 class DataFileSerializer(serializers.ModelSerializer):
     """Serializer for Data files."""
 
-    file = serializers.FileField(
-        write_only=True,
-        validators=[validate_data_file]
-    )
+    file = serializers.FileField(write_only=True)
```

Comment:
This validation was moved to the serializers `validate` method instead so we could access other fields on the record.